### ISSUE TITLE
Shaders not being seen

### ISSUE THREAD

**Issue by thededman** [Rakashazi/emu-ex-plus-alpha#9]
All the emulators are missing or can't read the HQ2x-v.txt or the Scale2x-v.txt for iOS. I had added them to the empty Xcode project with the other assets and executable. In the prior version 1.5.19  this was working. Possible a non-jailbreak issue or the executable looking for the wrong path? But was working in the prior version. I used the empty project added the assets and created the shaders and add the TXT files. The project was compiled and working but not the shaders.
![shader issues](https://cloud.githubusercontent.com/assets/5215424/6236326/da60c8b6-b6b7-11e4-892a-70bb5e6baf04.jpg)
![img_0008](https://cloud.githubusercontent.com/assets/5215424/6236330/e26a842a-b6b7-11e4-9a8c-2397c8b0d5e5.PNG)

**Comment by thededman** [comment]
I was reviewing the code: auto file = openAppAssetIO(makeFSPathStringPrintf("shaders/%s%s", useFallback ? "fallback-" : "", desc.vShaderFilename));

you have a extra /%s in there so can't see the folder.

**Comment by Rakashazi** [comment]
That code is correct, the extra %s is to select between the fallback versions of the shaders (only used on Android for old GPUs), and when using the regular versions it's just a blank string. It sounds like something else is going wrong in your case, but you can see the full file path the app is trying to open by watching the console output when the shader is loaded. Is it showing the correct base path into your app directory?

**Comment by thededman** [comment]
it seems I need to work on my Xcode skills. When I add the shaders to the project in Xcode, it's not adding to the build of the ipa. I was just trying tonight with a new empty project. I added the resources info files, icons, and the shaders folder. I added the group to the project, build and was not part of the ipa. Maybe my stupidity or missing something. Once I figure out I will document post under my fork to help others or just myself.
